docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,142 @@
+* customize my cluster with a pre-defined set of features and components.
+** scale my cluster horizontally and vertically, as my needs change.
+** choose the https://products.docs.vshn.ch/products/service_levels.html[service level^] of the instance.
+* delete the Cloud Kubernetes environment again.
+
+As a *Technical VSHNeer* I can:
+
+* see and configure the https://products.docs.vshn.ch/products/service_levels.html[service level^] of the instance.
+* automatically bill the service to the end-user according to the specified price model.
+
+== Iteration: Cluster Maintenance
+
+As an *End-User* I can:
+
+* schedule maintenance windows for my Cloud Kubernetes environment, during which updates, patches, or other changes can be applied automatically.
+* receive notifications in advance of any scheduled maintenance, allowing me to plan accordingly and to minimize any disruption to my applications.
+
+As a *Technical VSHNeer* I can:
+
+* monitor the status of the Cloud Kubernetes environment during the maintenance window, to ensure that the maintenance is operating as expected.
+
+== Iteration: Cluster Version Upgrade
+
+As an *End-User* I can:
+
+* schedule an automated upgrade of my Cloud Kubernetes environment to the next minor version.
+
+As a *Technical VSHNeer* I can:
+
+* trigger an automated upgrade of the Cloud Kubernetes environment to the next minor version.
+
+== Iteration: Cluster Backup and Restore
+
+As an *End-User* I can:
+
+* rely on automated backups of my Cloud Kubernetes environment configuration and cluster state.
+* schedule backups cluster state backups to run automatically, at a time and frequency that suits my needs.
+* configure data retention policies to ensure that backups are kept for the desired length of time.
+* quickly and easily restore my Cloud Kubernetes environment to a previous state, including all cluster configuration.
+* test restores in a test environment before performing a full restore, to ensure that the backup is fully restorable.
+
+== Iteration: Cluster-Level Metrics and Alerting
+
+As an *End-User* I can:
+
+* access a metrics dashboard to get an overview of the Cloud Kubernetes environment and health.
+
+As a *Technical VSHNeer* I can:
+
+* have a metrics service that will help me keep track of the health and performance of the Cloud Kubernetes environment.
+* enable alerting on pre-defined SLIs to VSHN so that VSHN can resolve incidents with priority.
+* see SLO reports so that I can get an overview of how well my Cloud Kubernetes environment performs.
+* access operations runbooks so that I can resolve upcoming alerts.
+* access a metrics dashboard to get an overview of the Cloud Kubernetes environment and health.
+
+== Iteration: End-User Metrics and Alerting
+
+As an *End-User* I can:
+
+* enable a service to automatically collect and aggregate metrics from my Cloud Kubernetes environment and applications, providing a unified view of all metrics.
```

Comment:
This looks like a duplication of line 69. But I guess the difference is one cluster vs several clusters. For the latter, I wonder if this is a feature of Cloud Kubernetes or a product on its own that can be used by everything VSHN provides?

Reply:
The differences between this line and line 69 is the context: Line 69 is in the "Cluster-Level Metrics" context and that one in the "End-User Metrics" context. Comparing that to OpenShift would mean: Here we're talking about the "User Workload Monitoring" feature and in line 69 about the standard cluster monitoring. Do you see anything which we could rephrase so that this gets more clear?

Reply:
I would suggest to maybe change the name of the iteration to "Applications Metrics and Alerting". In addition to what @corvus-ch mentioned, and as mentioned also before, I think that this could also be a product on its own.

Reply:
Yes, I agree, I renamed the iteration. For now, I'd like to keep it here as this is an important aspect for an end-user of the product, and I'm pretty sure we have to offer that by default not as an add-on.